### DISCUSSION TITLE
购买了一年的专业版，但是无法通过邮箱登陆，无法使用专业版，请给我解决

### DISCUSSION THREAD

**Discussion by WeiChenUm** [getlantern/lantern-forum-cn#756] [General]

**Comment by WeiChenUm** [comment]


**Reply by MaxMKMax** [reply]
@WeiChenUm 您好. 我已经帮您的问题提交上去了. 这两天请查看这个帖子或邮箱来获取激活码. 谢谢!

**Reply by WeiChenUm** [reply]
@Mike777Khoo 你好，问题反馈了这么多天了，为什么一直没有得到回复呢       验证码和激活码也一直没收到，解决个问题这么难吗

**Comment by whpublic2024** [comment]
他们现在不解决问题了吗？

**Reply by MaxMKMax** [reply]
@whpublic2024 您好! 我在另一个帖子回复您了. 您放心我们会帮您解决问题的. 我已经帮您的问题提交上去了. 这两天请查看这个帖子或邮箱来获取激活码. 谢谢!
